factory-helper.gemspec

```diff
@@ -0,0 +1,26 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'factory-helper/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "factory-helper"
+  spec.version       = FactoryHelper::VERSION
+  spec.authors       = ["theB264"]
+  spec.email         = ["public88878878887@hotmail.com"]
```

Comment:
Any interest in setting up a single email account to handle gem-related communication?